### ISSUE TITLE
WebView - XtraLayout.ResetLayout causes NullReferenceException

### ISSUE THREAD

**Issue by p0utine** [xenial-io/Xenial.Framework#56]
To repro, right-click any detail view layout that shows a web view control and click 'Reset Layout':

![image](https://user-images.githubusercontent.com/87371546/146201628-bf9ffe0e-616b-4cb8-b6af-fbe336e48d84.png)

Here's the exception:

```
	Type:       NullReferenceException
	Message:    Object reference not set to an instance of an object.
	Data:       0 entries
	Stack trace:

   at Microsoft.Web.WebView2.WinForms.WebView2.OnVisibleChanged(EventArgs e)
   at System.Windows.Forms.Control.SetVisibleCore(Boolean value)
   at System.Windows.Forms.Control.set_Visible(Boolean value)
   at DevExpress.XtraLayout.LayoutControlItem.OnControlVisibleChanged(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnVisibleChanged(EventArgs e)
   at Microsoft.Web.WebView2.WinForms.WebView2.OnVisibleChanged(EventArgs e)
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
	InnerException is null
```

In the demo I think we should disable `DevExpress.ExpressApp.Win.SystemModule.IModelWinLayoutManagerOptions.CustomizationEnabled`, but regardless we should fix this at the WebView editor level.

**Comment by biohazard999** [comment]
I think i resolved that issue with updating to the latest version of WebView2, but they had tons of breaking changes, so i reverted that change back then. 
We should investigate a update and see if this still occures.

**Comment by p0utine** [comment]
I confirm this is fixed in #50
